Fix for issue NVIDIA#4 - Allow scalar configs if function only takes one argument

Phoenix8215 opened the pull request:
Fix for issue NVIDIA#4 - Allow scalar configs if function only takes one argument

Alok-Joshi commented:
Please signoff your commits: https://github.com/NVIDIA/nsight-python/blob/main/Contributing.md#signing-your-work

Phoenix8215 commented:
> Thanks for this change. I was also working on this, and I have implemented a test and made doc updates. Overall this change looks good to me. Please address the comments.

Hi Alok. I’ve addressed your comments. Please let me know if anything else needs to be changed.

Phoenix8215 commented:
> @Phoenix8215 , I see two commits, one of which still does not have the sign off. Is it possible for you to rewrite the commit history the following way:
> 
> ```shell
> git reset --soft fd428ba6d0ad4fb503fbe4ccc98a1e1dd1bea317  #This is commit on the top of which you have made the changes
> 
> git commit -s -m <your_commit_message> 
> ```
> 
> And then you can do a force push:
> 
> ```shell
> git push -f <origin_name> <branch_name> 
> ```

Hi @Alok-Joshi, Thanks for pointing this out. I’ve rewritten the commit history using your instructions.
Please check if everything looks correct now.Thanks for your help!
